modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/HorizontalCardTag.java

```diff
@@ -73,15 +73,19 @@ public String getIcon() {
 
 	public String getTitle() {
 		String title = _title;
-
 		HorizontalCard horizontalCard = getHorizontalCard();
 
 		if ((_title == null) && (horizontalCard != null)) {
 			title = horizontalCard.getTitle();
+
+			if (horizontalCard.isLocalized()) {
+				title = LanguageUtil.get(
+					TagResourceBundleUtil.getResourceBundle(pageContext),
+					title);
+			}
```

Comment:
This still does not seem right. As I mentioned in https://github.com/liferay-frontend/liferay-portal/pull/2917#discussion_r1068014162, this `if` handles priority of tag attribute setting. It should not be related to localization. 

But I do understand why you did it this way! What we are missing is `localized` tag attribute of `<clay:horizontal-card>`, that is equivalent of model attribute. After a getter like this
```java
public boolean isLocalized() {
	String localized = _localized;

	HorizontalCard horizontalCard = getHorizontalCard();

	if ((_localized == null) && (horizontalCard != null)) {
		localized = horizontalCard.isLocalized();
	}

	return localized;
}
```

we can have it applied to title with something like:
```java
if ((_title == null) && (horizontalCard != null)) {
	title = horizontalCard.getTitle();
}

if (isLocalized()) {
	title = LanguageUtil.get(...);
}
```